content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -155,41 +155,14 @@ is reachable from the public internet.
 ### Authorization mode configuration {#choice-of-authz-config}
 
 You can configure the Kubernetes API server's authorizer chain using either
-[command line arguments](#using-flags-for-your-authorization-module) only or, as a beta feature,
-using a [configuration file](#using-configuration-file-for-authorization).
+a [configuration file](#using-configuration-file-for-authorization) only or
+[command line arguments](#using-flags-for-your-authorization-module).
```

Comment:
Do we also need to update the metadata for `StructuredAuthorizationConfiguration`?

Reply:
already updated in https://github.com/kubernetes/website/pull/48424